search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,354 @@
+
+The following specific CMIS QL fields are supported:
+
+* `cmis:parentId`
+* `cmis:objectId`
+* `cmis:objectTypeId`
+* `cmis:baseTypeId`
+* `cmis:contentStreamMimeType`
+* `cmis:contentStreamLength`
+
+The following CMIS QL comparison operators are supported:
+
+* `=`, `!=,` `<>`, `<`, `<=`, `>`, `>=`
+* `IN`, `NOT IN`, `LIKE`
+
+The following AFTS exact matches and patterns are supported:
+
+* `=<field>:term`
+* `=<field>:ter*`
+* `=<field>:*erm`
+
+### Supported for special fields in TMDQ using AFTS
+
+* PARENT
+* TYPE
+* ASPECT
+* EXACTTYPE
+* EXACTASPECT
+
+> **Note:** CMIS QL does not support any use of CONTAINS() using the database.
+
+Transactional Metadata Query and the Elasticsearch index queries are intended to support different use cases. They differ in queries and options that they support and in the results they generate with respect to collation and scoring.
+
+## Transactional metadata queries supported by database
+
+Use this information to understand the queries supported by the database.
+
+The Alfresco Full Text Search (FTS) query text can be used standalone or it can be embedded in CMIS-SQL using the `contains()` predicate function. The CMIS specification supports a subset of Alfresco FTS. For more information on search syntax, see [Alfresco Full Text Search Reference]({% link search-services/latest/using/index.md %}).
+
+**CMIS QL**
+
+The following object types and their sub-types are supported:
+
+* `cmis:document`
+
+    For example:
+
+    ```sql
+    select * from cmis:document
+    ```
+
+* `cmis:folder`
+
+    For example:
+
+    ```sql
+    select * from cmis:folder 
+    ```
+
+* Aspects
+
+    For example:
+
+    ```sql
+    select * from cm:dublincore 
+    ```
+
+## CMIS property data types
+
+The `WHERE` and `ORDER BY` clauses support the following property data types and comparisons:
+
+* `string`
+  * Supports all properties and comparisons, such as `=`, `<>`, `<`, `<=`, `>=`, `>`, `IN`, `NOT IN`, `LIKE`
+  * Supports ordering for single-valued properties
+    For example:
+
+    ```sql
+    select * from cmis:document where cmis:name <> 'fred' order by cmis:name
+    ```
+
+* `integer`
+  * Supports all properties and comparisons, such as `=`, `<>`, `<`, `<=`, `>=`, `>`, `IN`, `NOT IN`
+  * Supports ordering for single-valued properties
+
+* `double`
+  * Supports all properties and comparisons, such as `=`, `<>`, `<`, `<=`, `>=`, `>`, `IN`, `NOT IN`
+  * Supports ordering for single-valued properties
+* `float`
+  * Supports all properties and comparisons, such as `=`, `<>`, `<`, `<=`, `>=`, `>`, `IN`, `NOT IN`
+  * Supports ordering for single-valued properties
+* `boolean`
+  * Supports properties and comparisons, such as `=` and `<>`
+  * Supports ordering for single-valued properties
+* `id`
+  * Supports `cmis:objectId`, `cmis:baseTypeId`, `cmis:objectTypeId`, `cmis:parentId`, `=`, `<>`, `IN`, `NOT IN`
```

Comment:
```suggestion
  * Support for `cmis:objectId`, `cmis:baseTypeId`, `cmis:objectTypeId` and `cmis:parentId` fields
  * Support for comparison using `=`, `<>`, `IN`, `NOT IN`
```

Reply:
Done: search for   * Supports `cmis:objectId`, `cmis:baseTypeId`, `cmis:objectTypeId` and `cmis:parentId` fields
  * Support for comparisons, using `=`, `<>`, `IN`, `NOT IN`